optuna_dashboard/ts/trialFilter.ts

```diff
@@ -116,6 +116,29 @@ export const useFilteredTrials = (
     })
   }, [study?.trials, targets, filterComplete, filterPruned])
 
+export const getFilteredTrials = (
```

Comment:
Could you please `useFilteredTrials` to improve the re-rendering performance?

Reply:
I created the version of useFilteredTrials for multiple studies.